### DISCUSSION TITLE
BTT smart filament sensor

### DISCUSSION THREAD

**Discussion by xx1180** [mriscoc/Ender3V2S1#704] [Q&A]
i need to know if the BTT smart filament sensor can be used on the s1 pro instead of the stock sensor with this firmware and what would have to be changed

**Comment by Homer-S** [comment]
I'm using the BTT SFS on my Ender-3 v2 with Sprite Extruder Pro. Double check the pinouts between the sensor and board connection. In the printer menu I have selected "Motion" since this is a motion sensor not just a break sensor. It seems to be working OK. I suppose I should test it but I have managed to trigger the unload fine.
